package.json

```diff
@@ -28,7 +28,7 @@
     "typescript": "^2.3.3"
   },
   "jspm": {
-    "main": "src/js-form-validator.ts",
+    "main": "dist/js-form-validator.ts",
```

Comment:
@someyoungideas we don't want to be referencing the compiled output, we want that to be done on the client side so we can make use of the source files.